arangod/Aql/ShardLocking.h

```diff
@@ -59,14 +61,14 @@ class ShardLocking {
     SnippetInformation() : isRestricted(false), restrictedShards({}) {}
     // Flag if this snippet is restricted at all
     bool isRestricted;
-    // THe list of shards this snippet is restricted to.
-    // Invariant isRestricted == false => restrictedShards.empty()
-    // Invariant isRestricted == true => !restrictedShards.empty()
+    // The list of shards this snippet is restricted to.
+    // Invariant isRestricted == false <=>  restrictedShards.empty()
+    // Invariant isRestricted == true  <=> !restrictedShards.empty()
```

Comment:
so `isRestricted` and `restrictedShards.empty()` are equivalent?

Reply:
Yes. I didn't want to change the comment too much. ;)